src/execution/execute.js

```diff
@@ -322,18 +332,24 @@ function executeFields(
   exeContext: ExecutionContext,
   parentType: GraphQLObjectType,
   sourceValue: mixed,
+  exePath: Array<string | number>,
   fields: {[key: string]: Array<Field>}
 ): Object {
   let containsPromise = false;
 
   const finalResults = Object.keys(fields).reduce(
     (results, responseName) => {
       const fieldASTs = fields[responseName];
+
+      const childExePath = exePath.slice();
+      childExePath.push(responseName);
```

Comment:
Thanks! I will work on this feedback immediately and will follow up with
fixes. Hopefully before the lunch time.
2016년 6월 10일 (금) 오전 8:32, Lee Byron <notifications@github.com>님이 작성:

> In src/execution/execute.js
> <https://github.com/graphql/graphql-js/pull/396#discussion_r66633628>:
>
> >    fields: {[key: string]: Array<Field>}
> >  ): Object {
> >    let containsPromise = false;
> >
> >    const finalResults = Object.keys(fields).reduce(
> >      (results, responseName) => {
> >        const fieldASTs = fields[responseName];
> > +
> > +      const childExePath = exePath.slice();
> > +      childExePath.push(responseName);
>
> Just a minor detail that the changes to the body of this function should
> match executeFieldsSerially as much as possible.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/graphql/graphql-js/pull/396/files/a5ef8c71ea2fcd821b9bd3ab4345ebeb58d7272b#r66633628>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACfNPn5RKiKxWwvpQ_T30bi98iIk3G8sks5qKYOHgaJpZM4Ir9oK>
> .
>